### ISSUE TITLE
Invalid Manifest

### ISSUE THREAD

**Issue by pgbearensen** [alenichev/view_image#1]
I am desperate to add this function to chrome as i hate being forced to open images in new tabs but I can't install this from the store, I just get "invalid manifest" repeatedly. Even restarting chrome or reloading the store page do nothing. Any idea what's causing this?

**Comment by onemast** [comment]
Here you go:

https://chrome.google.com/webstore/detail/view-image/ggkpgcdaaginfhgoglcgaaicdkoicnhj

**Comment by pgbearensen** [comment]
@mic773 That link just takes me to the chrome webstore where, as I said, the install button simply gives me the "invalid manifest" popup every time I attempt to add this to chrome.
I don't understand if there is something else you want me to do with this link?

**Comment by onemast** [comment]
That's odd, it works for me. Try on a fresh Chrome installation maybe?
You can always download the source from here and try to install it manually.

**Comment by pgbearensen** [comment]
Thanks for you help so far mic773. Could you please explain how I would download the source and install it manually? I have the latest version of chrome (go to help > about chrome) so I don't know what is causing this.

**Comment by mic773** [comment]
I am not sure, but here is a guide:

https://medium.freecodecamp.org/how-to-create-and-publish-a-chrome-extension-in-20-minutes-6dc8395d7153

**Comment by onemast** [comment]
OK, so it seems that the new version of Google Chrome makes this extension invalid due to an old manifest file. I created a similar extension from scratch and you can get it here:

https://chrome.google.com/webstore/detail/view-this-image/poildibeoniaibomedledajlgfbohjda

**Comment by TigerCat-hu** [comment]
Thx, but I can't install it.
![view-this-image](https://user-images.githubusercontent.com/33827733/40966011-9aae5672-68af-11e8-8ec6-d54fea76365f.png)

**Comment by onemast** [comment]
OK use this one then it works perfectly (it's in English despite the misleading screenshot!):

EDIT: working now.

**Comment by TigerCat-hu** [comment]
Thx! Is this another extension?

**Comment by onemast** [comment]
Yes and it works just the same!

**Comment by TigerCat-hu** [comment]
Unfortunately not perfect, but thank you!

**Comment by onemast** [comment]
It has the exact same functionality as the original view_image extension in this git, am I missing something?

**Comment by TigerCat-hu** [comment]
Yes. Show an selected image on the current tab. Not the background image.

**Comment by OR94** [comment]
@pgbearensen I've had the same issue as you and I fixed it the following way.
Download the source code of the extension.
Extract the code to some folder.
Navigate to that folder.
Open manifest.json file in text editor and edit it to look like this:
`{
   "manifest_version": 2,
   "background": {
      "page": "background.html"
   },
   "default_locale": "en",
   "description": "__MSG_description__",
   "name": "__MSG_name__",
   "permissions": [ "contextMenus" ],
   "version": "1.3"
}`
Then, go to Chrome menu (3 dots), More tools, Extensions, turn on Developer mode, then Load unpacked extension from the folder you extracted the source code to.
That's it. Now everything should work!

**Comment by onemast** [comment]
Yes I know I did the same thing as I mentioned earlier, but using developer mode isn’t a good solution since it won’t sync between browsers on diffenet devices. The best solution is to use the alternative extension for now.

**Comment by mastef** [comment]
@mic773 @TigerCat-hu the reason he couldn't install your extension, is because you shared your "publish accepted" link. Downloading it through this link will work : https://chrome.google.com/webstore/detail/view-this-image/poildibeoniaibomedledajlgfbohjda

**Comment by TigerCat-hu** [comment]
@mastef Thank you very much! This is really awesome! :-)

**Comment by onemast** [comment]
@mastef you're right, I updated the link above as well. Thank you!
